core/modules/update/update.install

```diff
@@ -239,6 +239,35 @@ function update_update_1004() {
   $config->save();
 }
 
+/**
+ * Check installer.settings.json and fix it or remove it if needed.
+ */
+function update_update_1005() {
+  $config = config('installer.settings');
```

Comment:
What I was actually suggesting earlier was a comment about why this update hook exists, to explain why Installer code is in Update. Something like "Saving the update settings form created an empty copy of installer settings in earlier Backdrop versions. This update hook fixes this." Sorry to nitpick and not explain the nitpick  properly.

Reply:
OK, got it. Please check the latest version and let me know if this is what you had in mind.